### ISSUE TITLE
[ui] Need to add one more permission/role for GKE

### ISSUE THREAD

**Issue by leodotcloud** [rancher/rancher#14031]
Current text in add cluster page of GKE:

```
Create a Service Account with a JSON private key and provide the JSON here. See Google Cloud docs for more info about creating a service account. Three IAM roles are required: computer/viewer, project/viewer, kubernetes-engine/admin and service-account/user, more info on roles can be found here.
```

One more permission is needed: `compute.networks.list`.

**Comment by vincent99** [comment]
Really? Compute viewer should include listing everything.

**Comment by leodotcloud** [comment]
That's based on comments from a customer. I haven't gotten a chance to test it. Will have to ask @sangeethah to help us out on verifying this.

**Comment by develar** [comment]
![screen shot 2018-07-09 at 15 04 10](https://user-images.githubusercontent.com/350686/42452081-5e7c4ef8-8389-11e8-9748-0211e78869bc.png)

Yes, required. And it is strange since `compute.networks.list` in the `compute viewer` and I see that all specified roles are added to service account.

![screen shot 2018-07-09 at 15 08 02](https://user-images.githubusercontent.com/350686/42452306-f2fd019e-8389-11e8-8afe-059f231505a0.png)

**Comment by develar** [comment]
It seems it is sporadic GCE issue, third attempt to create service account was successful.

**Comment by superseb** [comment]
Can't reproduce yet, it works when adding the given IAM roles.

**Comment by develar** [comment]
@superseb Maybe only new users (compute engine is not enabled, k8s service not initialized) are affected. GCE UI is slow and back to 90 (wait several minutes, try a few minutes later and so on).

**Comment by deniseschannon** [comment]
@sangeethah i don't believe tihs is an actual issue, but can you test based on the permissions provided in the UI and give feedback?

**Comment by cjellick** [comment]
moved back to 2.1, since its to-test for reverification
